00-average/Average.java

```diff
@@ -0,0 +1,23 @@
+import java.util.Scanner;
+
+public class Average{
+  public static void main(String[] args) {
+    Scanner input = new Scanner(System.in);
+    int total = 0;
+    int manyGrade = 0;
+
+    System.out.println("How many class do you have? ");
+    manyGrade = input.nextInt();
+
+    for(int f = 0; f < manyGrade; f++){
+      System.out.println("What grade did you got? ");
+      int grade = input.nextInt();
+      total = (total + grade);
+    } //end for loop
+
+    int average = total / manyGrade;
+    System.out.println("Your average is " + average);
```

Comment:
will a decimal get printed in the case of {99, 100} (99.5 should be the output, i think 99 will be the actual output)?